src-test/org/etools/j1939_84/controllers/part1/Step02ControllerTest.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2020 Equipment & Tool Institute
+ */
+package org.etools.j1939_84.controllers.part1;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.Executor;
+
+import org.etools.j1939_84.bus.j1939.J1939;
+import org.etools.j1939_84.controllers.ResultsListener;
+import org.etools.j1939_84.controllers.TestResultsListener;
+import org.etools.j1939_84.model.PartResultFactory;
+import org.etools.j1939_84.model.VehicleInformation;
+import org.etools.j1939_84.modules.BannerModule;
+import org.etools.j1939_84.modules.DateTimeModule;
+import org.etools.j1939_84.modules.EngineSpeedModule;
+import org.etools.j1939_84.modules.ReportFileModule;
+import org.etools.j1939_84.modules.TestDateTimeModule;
+import org.etools.j1939_84.modules.VehicleInformationModule;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * The unit test for {@link Step02Controller}
+ *
+ * @author Marianne Schaefer (marianne.m.schaefer@gmail.com)
+ *
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class Step02ControllerTest {
+
+    @Mock
+    private BannerModule bannerModule;
+
+    private DateTimeModule dateTimeModule;
+
+    @Mock
+    private EngineSpeedModule engineSpeedModule;
+
+    @Mock
+    private Executor executor;
+
+    private Step02Controller instance;
+
+    @Mock
+    private J1939 j1939;
+
+    private TestResultsListener listener;
+
+    @Mock
+    private ResultsListener mockListener;
+
+    @Mock
+    private PartResultFactory partResultFactory;
+
+    @Mock
+    private ReportFileModule reportFileModule;
+
+    @Mock
+    private VehicleInformationModule vehicleInformationModule;
+
+    @Before
+    public void setUp() throws Exception {
+        listener = new TestResultsListener(mockListener);
+        dateTimeModule = new TestDateTimeModule();
+
+        instance = new Step02Controller(executor,
+                engineSpeedModule,
+                bannerModule,
+                dateTimeModule,
+                vehicleInformationModule,
+                partResultFactory);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        verifyNoMoreInteractions(executor,
```

Comment:
this needs to include the mockListener

Reply:
This resolution will include a couple of additional files as the inclusion of the mockListener in the verifyNoMoreInteractions was missed in previously written tests and those tests need to be updated as well.